server/acl_casbin_policy_developer.csv

```diff
@@ -10,6 +10,7 @@ p, DEVELOPER, /project/{id}/repository, GET
 p, DEVELOPER, /project/{id}/repository, POST
 p, DEVELOPER, /project/{id}/repository, PATCH
 p, DEVELOPER, /project/{id}/repository, DELETE
+p, DEVELOPER, /project/{id}/deployment, GET
```

Comment:
developer can also PATCH because this is the workspace role. And a workspace developer can still be a project owner.